### PR TITLE
Fix #261: Make `refRank` a proper index for histograms

### LINKED ISSUE #261: Add a row number to chart dataset from API
To help fix the exponential histograms we would like to add a row number column to the API for the chart data.

Currently `refRank` is the ordering that we get from the upstream data, which has no significance. We should change this to fit this need or remove it and name it something that better matches the goal, perhaps "index".

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: notebooks/import.py [hunk 1]
```diff
@@ -217,11 +217,13 @@ def create_points(collection_id, histogram):
            '(collection_id, bucket, proportion, count, rank) '
            'VALUES %s')
 
-    for rank, kv in enumerate(histogram.iteritems(), 1):
-        k, v = kv[0], kv[1].asDict()
+    rank = 0
+    for k in sorted(histogram.keys()):
```
Comment (sunahsuh) [inline notebooks/import.py]: Any particular reason to prefer manual iteration vs `enumerate`?

File: viewer/api/models.py [hunk 2]
```diff
@@ -10,7 +10,7 @@ def visible(self):
 
 class DataSet(models.Model):
     name = models.CharField(max_length=255, default='')
-    slug = models.CharField(max_length=50, unique=True)
+    slug = models.CharField(max_length=100, unique=True)
```
Comment (sunahsuh) [inline viewer/api/models.py]: Seems we might as well go for 255 -- it's not like we're taxing this db :)

File: notebooks/import.py (hunk 1 above)
Comment (robhudson) [reply notebooks/import.py]: Nope, just how it came out of my brain b/c of the change to sorting the keys. I'll update.

File: viewer/api/models.py (hunk 2 above)
Comment (robhudson) [reply viewer/api/models.py]: I thought I saw in normandy these were 50 characters but I can't find where that lives now. So yeah, I could blow it all the way out.

**Comment by robhudson** [comment]
Updated!

**Pull request merged**